chromium/debugging-rulesets.js

```diff
@@ -0,0 +1,20 @@
+/* global sendMessage */
+
+"use strict";
+
+document.getElementById("debugging-rulesets-form").addEventListener("submit", save_debugging_rulesets);
+
+sendMessage("get_option", { debugging_rulesets: "" }, item => {
+  let debugging_rulesets_textarea = document.getElementById("debugging-rulesets");
+  debugging_rulesets_textarea.value = item.debugging_rulesets;
+  debugging_rulesets_textarea.style.display = "block";
+});
+
+function save_debugging_rulesets(e){
+  e.preventDefault();
+  sendMessage("set_option", { debugging_rulesets: document.getElementById("debugging-rulesets").value }, () => {
+    const saved = document.getElementById("saved");
+    saved.style.display = "block";
+    setTimeout(() => { saved.style.display = "none" }, 1000);
```

Comment:
This code block is not getting called for me. I think it is because we are calling `sendResponse` asynchronously [here](https://github.com/Hainish/https-everywhere/blob/0ecb5b4f66a2b94adb8250f3eaf8084b5c41ef67/chromium/background.js#L558). But we are not returning `true` in that listener. See the note about "`return true`" [here](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/runtime/onMessage).

This is not a bug with this PR specifically, but it seems like a pretty bad thing. It is probably causing other problems. I'll fix it tomorrow and determine what the implications are.